### PR TITLE
Set up email notifications

### PR DESCRIPTION
**Pull request by yuna878**
## Overview

Set up email notifications

## Changes Made

- Put `notify_users()` under try-except block because errors from `push_notifications.py` don't get outputted on terminal for some reason. This should help with debugging in the future.

## Test Coverage

Manual testing with sendgrid & artificial data

### CODE REVIEW

File: envrc.template [hunk 1]
```diff
@@ -7,6 +7,6 @@ export IOS_CLIENT_ID=FILL_IN
 export DB_FILENAME=FILL_IN
 export FLASK_ENV=development
 export KEYFILE_PATH=FILL_IN
-export SEND_GRID_API_KEY=FILL_IN
+export SENDGRID_API_KEY=FILL_IN
```
Comment (alanna-zhou) [inline envrc.template:10]: nice :)

File: src/app/coursegrab/utils/push_notifications.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import json
 import jwt
 import time
-from app.coursegrab.utils.constants import ALGORITHM, ANDROID, IOS
+from app.coursegrab.utils.constants import ALGORITHM, ANDROID, IOS, EMAIL
```
Comment (alanna-zhou) [inline src/app/coursegrab/utils/push_notifications.py:4]: alphabetize E before I

File: src/app/coursegrab/utils/push_notifications.py [hunk 3]
```diff
@@ -126,4 +131,4 @@ def send_emails(section, emails):
         sendgrid_client = SendGridAPIClient(environ.get("SENDGRID_API_KEY"))
         sendgrid_client.send(message)
     except Exception as e:
-        print(e.body)
+        print(e)
```
Comment (alanna-zhou) [inline src/app/coursegrab/utils/push_notifications.py:134]: maybe `print("Error during creating payload", e)`? haha sometimes gets confusing at where these errors b coming from

**Review by alanna-zhou — APPROVED** [review]
yuna ur a beast 🥇

**Review by aliciaxw — APPROVED** [review]
v gooooood, agree with alanna

**Pull request merged**